internal/boxcli/generate/devcontainer_util.go

```diff
@@ -78,11 +79,17 @@ func CreateEnvrc(tmplFS embed.FS, path string) error {
 	if err != nil {
 		return errors.WithStack(err)
 	}
+	// setup values to pass to template
+	tmplData := struct {
+		EnvConfigFlagDisabled bool
+	}{
+		EnvConfigFlagDisabled: featureflag.EnvConfig.Disabled(),
+	}
```

Comment:
FYI, this could just be a map

Reply:
I know there are 2 ways to pass values, a struct or a map. I like the struct better because for each filed you specify the type. For map it has to be `map[string]any`. It's just preference.